From fa33ef9631877ec4c76ebe232acf9e1aab1e0cf7 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sun, 18 Sep 2022 18:28:30 +0200
Subject: [PATCH 6/6] copyright-format: Split the comment about license
 location into its own field

The reference of the GPL about its location on the filesystem is not
part of the GPL notice on the source, so it's confusing to mix this in
the same License field. Instead move it into its own Comment field, as
additional information.
---
 copyright-format-1.0.xml | 6 +++---
 debian/copyright         | 8 +++++---
 2 files changed, 8 insertions(+), 6 deletions(-)

diff --git a/copyright-format-1.0.xml b/copyright-format-1.0.xml
index 8dece57..22ff5f9 100644
--- a/copyright-format-1.0.xml
+++ b/copyright-format-1.0.xml
@@ -1238,7 +1238,7 @@ also delete it here.</programlisting>
  .
  You should have received a copy of the GNU General Public License
  along with this package; if not, see &lt;https://www.gnu.org/licenses/&gt;.
- .
+Comment:
  On Debian systems, the full text of the GNU General Public License
  version 2 can be found in the file '/usr/share/common-licenses/GPL-2'.</programlisting>
        </para>
@@ -1292,7 +1292,7 @@ License: GPL-2+
  .
  You should have received a copy of the GNU General Public License
  along with this package; if not, see &lt;https://www.gnu.org/licenses/&gt;.
- .
+Comment:
  On Debian systems, the full text of the GNU General Public License
  version 2 can be found in the file '/usr/share/common-licenses/GPL-2'.
  ]]></programlisting>
@@ -1379,7 +1379,7 @@ License: GPL-2+
  .
  You should have received a copy of the GNU General Public License
  along with this package; if not, see &lt;https://www.gnu.org/licenses/&gt;.
- .
+Comment:
  On Debian systems, the full text of the GNU General Public License
  version 2 can be found in the file '/usr/share/common-licenses/GPL-2'.]]></programlisting>
       </para>
diff --git a/debian/copyright b/debian/copyright
index d80be52..3881773 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -109,6 +109,8 @@ License: GPL-2+
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
  GNU General Public License for more details.
  .
- A copy of the GNU General Public License is available as
- /usr/share/common-licenses/GPL in the Debian distribution or on the World
- Wide Web at https://www.gnu.org/licenses/.
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <https://www.gnu.org/licenses/>.
+Comment:
+ On Debian systems, the complete text of the GNU General Public License
+ can be found in '/usr/share/common-licenses/GPL-2'.
-- 
2.37.2

